ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -339,8 +414,17 @@ def _write_failure_state(reason):
     return reason
 
 
+def _read_addr():
+    return machine.mem32[REG_RTC_ADDR]
+
+
+def _write_addr(addr):
+    machine.mem32[REG_RTC_ADDR] = addr
+    return addr
```

Comment:
The return value is unused.

Reply:
True, I was copying the pattern of _write_failure_state above but yeah it's return is used at one point. _write_state above that doesn't return anything. So it wasn't really a pattern at all...